scheduler/scheduler-node/src/main/java/org/ow2/proactive/scheduler/task/client/SchedulerNodeClient.java

```diff
@@ -0,0 +1,436 @@
+        client.setSession(sid);
+    }
+
+    public String getSession() {
+        return client.getSession();
+    }
+
+    public boolean isJobFinished(JobId jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.isJobFinished(jobId);
+    }
+
+    public boolean isJobFinished(String jobId) throws NotConnectedException, UnknownJobException, PermissionException {
+        return client.isJobFinished(jobId);
+    }
+
+    public JobResult waitForJob(JobId jobId, long timeout) throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
+        return client.waitForJob(jobId, timeout);
+    }
+
+    public JobResult waitForJob(String jobId, long timeout) throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
+        return client.waitForJob(jobId, timeout);
+    }
+
+    public boolean isTaskFinished(String jobId, String taskName) throws UnknownJobException, NotConnectedException, PermissionException, UnknownTaskException {
+        return client.isTaskFinished(jobId, taskName);
+    }
+
+    public TaskResult waitForTask(String jobId, String taskName, long timeout) throws UnknownJobException, NotConnectedException, PermissionException, UnknownTaskException, TimeoutException {
+        return client.waitForTask(jobId, taskName, timeout);
+    }
+
+    public List<JobResult> waitForAllJobs(List<String> jobIds, long timeout) throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
+        return client.waitForAllJobs(jobIds, timeout);
+    }
+
+    public Map.Entry<String, JobResult> waitForAnyJob(List<String> jobIds, long timeout) throws NotConnectedException, UnknownJobException, PermissionException, TimeoutException {
+        return client.waitForAnyJob(jobIds, timeout);
+    }
+
+    public Map.Entry<String, TaskResult> waitForAnyTask(String jobId, List<String> taskNames, long timeout) throws UnknownJobException, NotConnectedException, PermissionException, UnknownTaskException, TimeoutException {
+        return client.waitForAnyTask(jobId, taskNames, timeout);
+    }
+
+    public List<Map.Entry<String, TaskResult>> waitForAllTasks(String jobId, List<String> taskNames, long timeout) throws UnknownJobException, NotConnectedException, PermissionException, UnknownTaskException, TimeoutException {
+        return client.waitForAllTasks(jobId, taskNames, timeout);
+    }
+
+    public boolean pushFile(String spacename, String pathname, String filename, String file) throws NotConnectedException, PermissionException {
+        return client.pushFile(spacename, pathname, filename, file);
+    }
+
+    public void pullFile(String space, String pathname, String outputFile) throws NotConnectedException, PermissionException {
+        client.pullFile(space, pathname, outputFile);
+    }
+
+    public boolean deleteFile(String space, String pathname) throws NotConnectedException, PermissionException {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1161)